tests/bgp/test_bgp_speaker.py

```diff
@@ -148,6 +225,20 @@ def common_setup_teardown(duthosts, rand_one_dut_hostname, ptfhost, localhost, t
         time.sleep(2)
         duthost.command("ip route add %s/32 dev %s" % (ip.ip, mg_facts['minigraph_vlan_interfaces'][0]['attachto']))
 
+    if ("2byte" in request.param) or ("4byte" in request.param):
+        bgp_speaker_4byteasn="65536"
```

Comment:
In case `"2byte" in request.param`, variable `bgp_speaker_asn` still will get value `65536`. I think it is a 4 bytes scenario. Is the logic correct here?